baselines/dom.generated.d.ts

```diff
@@ -3696,49 +4450,53 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
-    onwebkitfullscreenerror: (this: Element, ev: Event) => any;
+    onariarequest: ((this: Element , ev: Event) => any) | null;
+    oncommand: ((this: Element , ev: Event) => any) | null;
+    ongotpointercapture: ((this: Element , ev: PointerEvent) => any) | null;
+    onlostpointercapture: ((this: Element , ev: PointerEvent) => any) | null;
+    onmsgesturechange: ((this: Element , ev: Event) => any) | null;
+    onmsgesturedoubletap: ((this: Element , ev: Event) => any) | null;
+    onmsgestureend: ((this: Element , ev: Event) => any) | null;
+    onmsgesturehold: ((this: Element , ev: Event) => any) | null;
+    onmsgesturestart: ((this: Element , ev: Event) => any) | null;
+    onmsgesturetap: ((this: Element , ev: Event) => any) | null;
+    onmsgotpointercapture: ((this: Element , ev: Event) => any) | null;
+    onmsinertiastart: ((this: Element , ev: Event) => any) | null;
+    onmslostpointercapture: ((this: Element , ev: Event) => any) | null;
+    onmspointercancel: ((this: Element , ev: Event) => any) | null;
+    onmspointerdown: ((this: Element , ev: Event) => any) | null;
+    onmspointerenter: ((this: Element , ev: Event) => any) | null;
+    onmspointerleave: ((this: Element , ev: Event) => any) | null;
+    onmspointermove: ((this: Element , ev: Event) => any) | null;
+    onmspointerout: ((this: Element , ev: Event) => any) | null;
+    onmspointerover: ((this: Element , ev: Event) => any) | null;
+    onmspointerup: ((this: Element , ev: Event) => any) | null;
+    ontouchcancel: ((this: Element , ev: Event) => any) | null;
+    ontouchend: ((this: Element , ev: Event) => any) | null;
+    ontouchmove: ((this: Element , ev: Event) => any) | null;
+    ontouchstart: ((this: Element , ev: Event) => any) | null;
+    onwebkitfullscreenchange: ((this: Element , ev: Event) => any) | null;
+    onwebkitfullscreenerror: ((this: Element , ev: Event) => any) | null;
     outerHTML: string;
     readonly prefix: string | null;
     readonly scrollHeight: number;
     scrollLeft: number;
     scrollTop: number;
     readonly scrollWidth: number;
-    readonly tagName: string;
-    readonly assignedSlot: HTMLSlotElement | null;
-    slot: string;
     readonly shadowRoot: ShadowRoot | null;
-    getAttribute(name: string): string | null;
+    slot: string;
+    readonly tagName: string;
+    attachShadow(shadowRootInitDict: ShadowRootInit): ShadowRoot;
+    closest<K extends keyof HTMLElementTagNameMap>(selector: K): HTMLElementTagNameMap[K] | null;
+    closest<K extends keyof SVGElementTagNameMap>(selector: K): SVGElementTagNameMap[K] | null;
+    closest(selector: string): Element | null;
+    getAttribute(qualifiedName: string): string | null;
     getAttributeNS(namespaceURI: string, localName: string): string;
     getAttributeNode(name: string): Attr | null;
     getAttributeNodeNS(namespaceURI: string, localName: string): Attr | null;
     getBoundingClientRect(): ClientRect | DOMRect;
```

Comment:
Probably i missed something, but i can not find the definition of ClientRect. This is available in TS master:
https://github.com/Microsoft/TypeScript/blob/bb05122a50fa294ad9ab8da8b19ecbd76fe193e1/lib/lib.dom.d.ts#L1691
I think many are using the ClientRect definition, so this should survive.

Reply:
not sure i understand the question. the definition of `ClientRect` is at https://github.com/Microsoft/TSJS-lib-generator/blob/8d0ab24514f187d005028c8bd77c0439bd36dfe6/baselines/dom.generated.d.ts#L2892

Reply:
The was no change, so it was collapsed in the github webinterface. I just found it by myself. Sorry for the noise.